app/models/Comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_comment_counter
+
+  def update_comment_counter
+    post.update(comments_counter: post.comments.size)
+  end
```

Comment:
## Optional
- [ ] This method should be private after adding [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save`